tasks/gulp/compile-assets.js

```diff
@@ -138,6 +143,19 @@ gulp.task('scss:compile', () => {
         })
       ]))
       .pipe(gulp.dest(taskArguments.destination + '/'))
+
+    compileFullPageStyles = gulp.src(compileFullPageExampleStylesheets)
+      .pipe(plumber(errorHandler))
+      .pipe(sass())
+      .pipe(rename(function (path) {
+        path.basename = path.dirname
+        path.dirname = ''
+      }))
+      .pipe(gulp.dest(taskArguments.destination + '/full-page-examples/'))
+  }
+
+  if (isPublic) {
+    return compileFullPageStyles
```

Comment:
@36degrees I wasn't sure how this worked either... I've done a bit of a refactor now so that the gulp tasks hopefully make a bit more sense - do you want to take another look?